src/actions/pkg.cr

```diff
@@ -2,26 +2,114 @@ module DppmRestApi::Actions::Pkg
   extend self
   ALL_PKGS = ""
   ONE_PKG  = "/:id"
+
+  def output_config_for(package, to builder)
+    builder.field package.package do
+      builder.object do
+        package.pkg_file.config_vars.try &.each_key do |key|
+          builder.field key, package.get_config key
```

Comment:
Use https://github.com/DFabric/dppm/blob/master/src/prefix/app.cr#L240 instead, else you'll miss library conf files like `/conf/php-fpm.ini`

Reply:
I don't understand, this is for the package. The application configurations are a different route. Or am I misunderstanding how the app/pkg structure works?

Reply:
Yes sorry, I meant using `Pkg#each_config_key`, else dependencies configuration will be missed (php-fpm, in the future JVM).